timelock-server/src/stressTest/java/com/palantir/atlasdb/timelock/AbstractStressTest.java

```diff
@@ -119,12 +100,7 @@ private static void assertNumberOfThreadsReasonable(int startingThreads, int thr
         }
     }
 
-    private void abandonLeadershipPaxosModeAgnosticTestIfRunElsewhere() {
-        Assume.assumeTrue(cluster == FIRST_CLUSTER);
-        Assume.assumeFalse(cluster.isDbTimelock()); // We will never test only DB timelock when releasing.
-    }
```

Comment:
The tests were not running due to this issue. Previously, stress tests did not have their own CircleCI container, so only one parameter was run to save time. Later, someone refactored the stress tests to have a separate CircleCI container but forgot to update these lines. Now, we can enable all parameters and it takes 7 minutes, which is acceptable since we have CircleCI tasks that take longer. (Note that sometime we were not running any parameters at all because FIRST_CLUSTER is also DB timelock)